examples/normalizer/Normalize.py

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/env python
+
+import sys
+import csv
+from trigger.cmds import ReactorlessCommando
+from twisted.python import log
+from twisted.internet import reactor
+from Router import Router
+import jsonpickle
+
+
+class getRouterDetails(ReactorlessCommando):
+    commands = Router.showCommands
+
+    def select_next_device(self, jobs=None):
+        """
+        Select another *reachable* device.
+        """
+        if jobs is None:
+            jobs = self.jobs
+
+        next_device = jobs.pop()
+        log.msg('Selecting next device: %s' % next_device)
+
+        # If I ping, return me.
+        if next_device.is_reachable():
+            log.msg('PING [SUCCESS]: %s' % next_device)
+            return next_device
+        # Otherwise return None and store me as an error.
+        else:
+            msg = 'PING [FAILURE]: %s' % next_device
+            log.msg(msg)
+            print "Failed to ping host {}".format(next_device)
+            if self.verbose:
+                print msg
+            return None
+
+    def errback(self, failure, device):
+        print "Error in getRouterDetails for device {}\n{}".format(
+            device,
+            failure.getTraceback()
+        )
+
+
+def validateRouterDetails(result):
+    print "Validating router details"
+    devicesToCorrect = []
+
+    for device, results in result.items():
+        routers[device].results = results
+        routers[device].validate()
+        routers[device].normalize()
+        if routers[device].normalizeRequired:
+            devicesToCorrect.append(device)
+
+    return devicesToCorrect or None
+
+
+class normalizeRouters(ReactorlessCommando):
+    def to_cisco(self, dev, commands=None, extra=None):
+        dev_commands = routers[dev.nodeName].commands
+        # self.commands = dev_commands
+        # print "Device {}: Executing Commands:\n{}".format(
+        #       dev.nodeName, dev_commands)
```

Comment:
If these aren't being used, please delete them! Or uncomment them if you think they have value.